README.md

```diff
@@ -12,40 +12,18 @@ This is _work in progress_. The final config will be shared prior to launch date
 * Status: _Planned_ ([#3](https://github.com/eth-clients/holesky/issues/3))
 * Flag: `--holesky`
 * Flavor: Permissionless (Proof-of-Stake), _to replace Goerli_
-* Launch Date: **Sept/15, 2023, 14:00 UTC**
-  * Epoch time: 1694786400
+* Launch Date: **Sept/28, 2023, 12:00 UTC**
```

Comment:
This time suits the Australians & Kiwis a lot better than 14:00UTC. And Thursday is better than Friday as it gives a bit of a buffer to fix things before the weekend :+1: